Documentation/project-docs/contributing.md

```diff
@@ -26,6 +26,17 @@ Pull Requests
 * **DO** refer to any relevant issues, and include [keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) that automatically close issues when the PR is merged.
 * **DO** tag any users that should know about and/or review the change.
 * **DO** ensure each commit successfully builds.  The entire PR must pass all tests in the Continuous Integration (CI) system before it'll be merged.
-* **DO** address PR feedback in an additional commit(s) rather than ammending the existing commits, and only rebase/squash them when necessary.  This makes it easier for reviewers to track changes. If necessary, squashing should be handled by the merger using the ["squash and merge"](https://github.com/blog/2141-squash-your-commits) feature, and should only be done by the contributor upon request.
+* **DO** address PR feedback in an additional commit(s) rather than amending the existing commits, and only rebase/squash them when necessary.  This makes it easier for reviewers to track changes.
+* **DO** assume that ["Squash and Merge"](https://github.com/blog/2141-squash-your-commits) will be used to merge your commit unless you request otherwise in the PR. Reasons may include (but not limited to)
```

Comment:
I disagree:
- Maintainers look at this, too, and there should be consistent guidance applied.
- Contributors obviously care (i.e. you) about when/why commits are being merged or squashed, and they should be able to look here to understand the general guidance.
- If contributors are likely to disagree, then that's good reason to list examples here so that we can get to place where disagreement is minimized, at which point we should have a better policy than when we started.